src/editor/codemirror/language-server/documentation.ts

```diff
@@ -112,25 +127,63 @@ export const wrapWithDocumentationButton = (
   docsAndActions.style.justifyContent = "space-between";
   docsAndActions.appendChild(child);
 
-  const anchor = docsAndActions.appendChild(document.createElement("a"));
-  anchor.href = "";
-  anchor.style.fontSize = "var(--chakra-fontSizes-sm)";
-  anchor.style.color = "var(--chakra-colors-brand-600)";
-  anchor.textContent = intl.formatMessage({ id: "help" });
-  anchor.style.display = "block";
-  anchor.style.margin = "0";
-  anchor.style.marginRight = "-0.5rem";
-  anchor.style.padding = "0.5rem";
-  anchor.style.alignSelf = "flex-end";
-  anchor.onclick = (e) => {
-    e.preventDefault();
-    document.dispatchEvent(
-      new CustomEvent("cm/openDocs", {
-        detail: {
-          id,
-        },
-      })
-    );
-  };
+  const actionsContainer = docsAndActions.appendChild(
+    document.createElement("div")
+  );
+  actionsContainer.style.display = "flex";
+  actionsContainer.style.justifyContent = "flex-end";
+  if (referenceLink) {
+    const refAnchor = createStyledAnchorElement();
+    refAnchor.style.marginRight = "0.5rem";
+    refAnchor.textContent = intl.formatMessage({ id: "help" });
+    refAnchor.onclick = (e) => {
+      e.preventDefault();
+      document.dispatchEvent(
+        new CustomEvent("cm/openDocs", {
+          detail: {
+            tab: "reference",
+            id: referenceLink,
+          },
+        })
+      );
+    };
+    actionsContainer.appendChild(refAnchor);
+  }
+  // We don't have documentation for builtins yet,
+  // so there is nothing to link to.
+  if (id.split(".")[0] !== "builtins") {
+    const apiAnchor = createStyledAnchorElement();
+    apiAnchor.textContent = intl.formatMessage({ id: "api-tab" });
+    apiAnchor.onclick = (e) => {
+      e.preventDefault();
+      document.dispatchEvent(
+        new CustomEvent("cm/openDocs", {
+          detail: {
+            tab: "api",
+            id,
+          },
+        })
+      );
+    };
+    actionsContainer.appendChild(apiAnchor);
+  }
   return docsAndActions;
 };
+
+export const getLinkToReference = (
+  id: string,
+  apiReferenceMap: ApiReferenceMap
+): string | undefined => {
+  const idSegments = id.split(".");
+  const pythonModuleName = idSegments[0];
+  const apiId = idSegments.slice(1).join(".");
```

Comment:
Please can you consider having some common utility for Python API name manipulation?

Reply:
[`7f2a572`](https://github.com/microbit-foundation/python-editor-next/pull/806/commits/7f2a5729e789dadbb060dcb74fbfc5f60b4ad681).